<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>YaIRC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/yairc.png</normaloff>:/icons/icons/yairc.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QTextEdit" name="textEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
        <property name="placeholderText">
         <string>Aucun channel n'est selectionné</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Sujet</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QListView" name="listView_2">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QListView" name="listView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>YaIRC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>Message</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuYaIRC">
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="title">
     <string notr="true">&amp;YaIRC</string>
    </property>
    <addaction name="actionServeurs"/>
    <addaction name="separator"/>
    <addaction name="actionFermer_l_onglet"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter_YaIRC"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>&amp;Affichage</string>
    </property>
    <addaction name="actionSujet_du_channel"/>
    <addaction name="actionListe_des_utilisateurs"/>
    <addaction name="separator"/>
    <addaction name="actionPlein_cran"/>
   </widget>
   <widget class="QMenu" name="menuServeur">
    <property name="title">
     <string>&amp;Serveur</string>
    </property>
    <addaction name="actionConnecter"/>
    <addaction name="actionD_connecter"/>
    <addaction name="actionRe_joindre_un_channel"/>
    <addaction name="separator"/>
    <addaction name="action_Liste_des_channels"/>
   </widget>
   <widget class="QMenu" name="menuParam_tres">
    <property name="title">
     <string>&amp;Paramètres</string>
    </property>
    <addaction name="action_Pr_f_rences"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="action_A_propos"/>
   </widget>
   <addaction name="menuYaIRC"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuServeur"/>
   <addaction name="menuParam_tres"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionServeurs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/server.png</normaloff>:/icons/icons/server.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Serveurs...</string>
   </property>
  </action>
  <action name="actionFermer_l_onglet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/close.png</normaloff>:/icons/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fermer l'onglet</string>
   </property>
  </action>
  <action name="actionQuitter_YaIRC">
   <property name="icon">
    <iconset theme="exit" resource="resources.qrc">
     <normaloff>:/icons/icons/quit.png</normaloff>:/icons/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter YaIRC</string>
   </property>
  </action>
  <action name="actionMenus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barre de &amp;menu</string>
   </property>
  </action>
  <action name="actionSujet_du_channel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barre de &amp;sujet</string>
   </property>
  </action>
  <action name="actionListe_des_utilisateurs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Liste des &amp;utilisateurs</string>
   </property>
  </action>
  <action name="actionPlein_cran">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/fullscreen.png</normaloff>:/icons/icons/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Plein écran</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionD_connecter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/disconnect.png</normaloff>:/icons/icons/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Déconnecter</string>
   </property>
  </action>
  <action name="actionConnecter">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/reconnect.png</normaloff>:/icons/icons/reconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connecter</string>
   </property>
  </action>
  <action name="actionRe_joindre_un_channel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/join_chan.png</normaloff>:/icons/icons/join_chan.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;joindre un channel</string>
   </property>
  </action>
  <action name="action_Liste_des_channels">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/list_chans.png</normaloff>:/icons/icons/list_chans.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Liste des channels</string>
   </property>
  </action>
  <action name="action_A_propos">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/about.png</normaloff>:/icons/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;A propos...</string>
   </property>
  </action>
  <action name="action_Pr_f_rences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Préférences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuitter_YaIRC</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionListe_des_utilisateurs</sender>
   <signal>toggled(bool)</signal>
   <receiver>listView_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1010</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSujet_du_channel</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
